plaid/institutions.go

```diff
@@ -122,25 +123,32 @@ type SearchInstitutionsResponse struct {
 // See https://plaid.com/docs/api/#institutions-by-id.
 func (c *Client) GetInstitutionByID(
 	id string,
+	countryCodes []string,
 ) (resp GetInstitutionByIDResponse, err error) {
-	return c.GetInstitutionByIDWithOptions(id, GetInstitutionByIDOptions{})
+	return c.GetInstitutionByIDWithOptions(id, countryCodes, GetInstitutionByIDOptions{})
 }
 
 // GetInstitutionByIDWithOptions returns information for a single institution given an ID.
 // See https://plaid.com/docs/api/#institutions-by-id.
 func (c *Client) GetInstitutionByIDWithOptions(
 	id string,
+	countryCodes []string,
 	options GetInstitutionByIDOptions,
 ) (resp GetInstitutionByIDResponse, err error) {
 	if id == "" {
 		return resp, errors.New("/institutions/get_by_id - institution id must be specified")
 	}
 
+	if len(countryCodes) == 0 {
```

Comment:
Instead of doing one-off errors like this, I wonder if we should just rely on the API to return an API error saying the same thing so that we can keep all of our validation in one place. Wdyt?

Reply:
Yea I was kind of confused by this. I think it probably makes more sense to remove these and leave it up to the API to handle validation.